DrawerKit/DrawerKit/Internal API/PresentationController+Gestures.swift

```diff
@@ -48,9 +40,6 @@ extension PresentationController {
                                                               configuration: configuration)
             animateTransition(to: endingState)
 
-        case .cancelled:
-            animateTransition(to: lastDrawerState)
```

Comment:
And, likewise, I don't understand the reason for removing this.

Reply:
`cancelled` is treated as `completed` now, and picks the next state based on current drawer state plus velocity.

Reply:
So, if the user starts dragging the drawer up then changes his mind, the drawer continues up anyway? That doesn't sound right to me.